src/qs_efield_berry.F

```diff
@@ -248,9 +248,9 @@ SUBROUTINE qs_efield_derivatives(qs_env, just_energy, calculate_forces)
       use_virial = virial%pv_availability .AND. (.NOT. virial%pv_numer)
       use_virial = use_virial .AND. calculate_forces
       ! disable stress calculation
-      IF (use_virial) THEN
-         CPABORT("Stress tensor for periodic E-field not implemented")
-      END IF
+      !IF (use_virial) THEN ! [pcazade] enabled the stress tensor
+      !   CPABORT("Stress tensor for periodic E-field not implemented")
+      !END IF
```

Comment:
please remove completely

Reply:
Reading the code it seems that the stress tensor for periodic E-field is actually implemented. Juerg told me that it is disabled because not sufficiently tested. I needed this information. It has nothing to do with space group symmetry though. Should I put it back?

Reply:
Either that, or switch the `CPABORT` to a `CPWARN` with an appropriate message, @juerghutter what do you think?